src/utils/version.js

```diff
@@ -26,27 +26,26 @@ const version = {
    * @memberof Version
    * @inner
    */
-  checkApiVersion() {
-    const newPromise = new Promise((resolve, reject) => {
-      const libPromise = hathorLib.version.checkApiVersion();
-      libPromise.then((data) => {
-        // Update version allowed in redux
-        store.dispatch(isVersionAllowedUpdate({allowed: hathorLib.helpers.isVersionAllowed(data.version, hathorLib.constants.MIN_API_VERSION)}));
-        // Set network in lib to use the correct address byte
-        let network;
-        if (data.network === 'mainnet') {
-          network = 'mainnet';
-        } else {
-          // Can we assume it will be testnet? I think it's safe
-          network = 'testnet';
-        }
-        helpers.updateNetwork(network);
-        resolve({...data, network});
-      }, (error) => {
-        reject();
-      });
-    });
-    return newPromise;
+  async checkApiVersion(wallet) {
```

Comment:
It would be important to describe this parameter on the JSDocs.

Reply:
Done in 84545042b377c6827ae98f8defff3babb88aa0b1